### DISCUSSION TITLE
Vowel.com research

### DISCUSSION THREAD

**Discussion by madhuroopa** [SartajBhuvaji/Resonate#5] [General]
Research [Vowel](https://www.vowel.com/)

Unique Features
Transcribes meetings live and allows users to search the transcript for specific keywords or topics.
Automatically generates summaries of meetings, highlighting key decisions and action items.
An AI-powered assistant that helps with tasks like scheduling meetings, taking notes, and reminding attendees of action items.
Enables translation of spoken words in real-time for multilingual participants.
Provides insights into the emotional tone of the conversation
What's similar ?
our project and vowel.com both leverage AI to generate summaries of meetings, aiming to extract key info

Differences:
Offline Meeting Summarization: Focuses on summarizing pre-recorded meetings, offering flexibility for asynchronous analysis and review.
Emphasizes storage of embeddings, potentially enabling efficient retrieval, similarity searches, and future analysis beyond summarization.
Data: We require uploading video and audio, while Vowel likely operates on live streams or processed recordings.
Output product: project focuses on building a bot, suggesting interactive summarization. Vowel primarily generates textual summaries.

LLM Used

Embedding Used - same here

Do they finetune their LLM?
They did not disclose about the LLM's used anywhere, sent an email to them will update once I receive any answer

Anything that we can adapt?

Train our LLM to identify and prioritize essential information like:
Decisions made during the meeting.
Action items assigned to participants.
Key arguments or disagreements discussed.
Important deadlines or next steps.
Utilize Named Entity Recognition (NER) and other techniques to extract relevant entities (people, dates, etc.) from the conversation.
Extract speaker identification, sentiment analysis etc from the video or audio stream- we can think of later these are jus addons
Combining audio and video features with textual content for good textual summaries ( as we thought of during our discussions)

**Comment by SartajBhuvaji** [comment]
Wow thats an insightful review!
